examples/pitch_plots/plot_arcs_around_goals.py

```diff
@@ -0,0 +1,42 @@
+"""
+==========================
+Plotting arcs around goals
+==========================
+
+This example shows how to plot arcs around goals at specified distances on various pitch types.
+"""
+from mplsoccer import Pitch, VerticalPitch
+import matplotlib.pyplot as plt
+
+
+def plot_arcs_around_goals_on_pitches(pitch_cls):
+    fig, axes = plt.subplots(4, 2, figsize=(12, 14))
+    axes = axes.ravel()
+    pitch_types = ['statsbomb', 'opta', 'tracab', 'skillcorner', 'wyscout',
```

Comment:
This is a simplified version of `examples/pitch_setup/plot_compare_pitches.py` which I was using to make sure I dealt with the subtleties of the different coordinate systems and aspect ratios.